modules/installation-azure-user-defined-routing.adoc

```diff
@@ -1,75 +1,79 @@
-is required to use outbound routing when installing a cluster; the installation
-program is not responsible for configuring this.
+You can configure user-defined routing by modifying parameters in the `install-config.yaml` file before installing your cluster. A pre-existing VNet is required to use outbound routing when installing a cluster; the installation program is not responsible for configuring this.
 
-When configuring a cluster to use user-defined routing, the installation program
-does not create the following resources:
+When configuring a cluster to use user-defined routing, the installation program does not create the following resources:
 
 * Outbound rules for access to the internet.
 * Public IPs for the public load balancer.
-* Kubernetes Service object to add the cluster machines to the public load
-balancer for outbound requests.
+* Kubernetes Service object to add the cluster machines to the public load balancer for outbound requests.
 
 You must ensure the following items are available before setting user-defined
 routing:
 
 * Egress to the internet is possible to pull container images, unless using an
 {product-registry} mirror.
 * The cluster can access Azure APIs.
-* Various allowlist endpoints are configured. You can reference these endpoints
-in the _Configuring your firewall_ section.
+* Various allowlist endpoints are configured. You can reference these endpoints in the _Configuring your firewall_ section.
+
+There are several pre-existing networking setups that are supported for internet access using user-defined routing.
+
+ifdef::restricted[]
+[discrete]
+== Restricted cluster with no internet access
+
+You can install a restricted network that restricts all access to the internet, except the Azure API. This is accomplished by mirroring the release image registry locally. Your cluster must have access to the following:
+
+* An {product-registry} mirror that allows for pulling container images
+* Access to Azure APIs
+
+[discrete]
+== Restricted cluster with Azure Firewall
+
+You can use Azure Firewall to restrict the outbound routing for the Virtual Network (VNet) that is used to install the {product-title} cluster. For more information, see link:https://learn.microsoft.com/en-us/azure/aks/egress-outboundtype#deploy-a-cluster-with-outbound-type-of-udr-and-azure-firewall[providing user-defined routing with Azure Firewall]. You can create a {product-title} cluster in a restricted network by using VNet with Azure Firewall and configuring the user-defined routing.
 
-There are several pre-existing networking setups that are supported for internet
-access using user-defined routing.
+[IMPORTANT]
+====
+If you are using Azure Firewall for restricting internet access, you must set the `publish` field to `Internal` in the `install-config.yaml` file. Because, link:https://learn.microsoft.com/en-us/azure/firewall/integrate-lb[Azure Firewall does not work properly with Azure public load balancers].
```

Comment:
```suggestion
If you are using Azure Firewall for restricting internet access, you must set the `publish` field to `Internal` in the `install-config.yaml` file. This is because link:https://learn.microsoft.com/en-us/azure/firewall/integrate-lb[Azure Firewall does not work properly with Azure public load balancers].
```